systems/metrics/exporter/pkg/config.go

```diff
@@ -29,7 +30,7 @@ type Config struct {
 	Queue             *config.Queue     `default:"{}"`
 	Service           *config.Service   `default:"{}"`
 	ExporterHost      string            `default:"localhost"`
-	Org               string            `default:"40987edb-ebb6-4f84-a27c-99db7c136100"`
+	Org               string            `default:"8c6c2bec-5f90-4fee-8ffd-ee6456abf4fc"`
```

Comment:
Lets remove this default org, It should be empty by default so that we can get appropriate error if its not set by configs.